js/client/modules/@arangodb/testsuites/ldap.js

```diff
@@ -54,7 +54,7 @@ const testPaths = {
   'ldapsearchplaceholder': [tu.pathForTesting('client/authentication')],
   'ldaprolesimple': [tu.pathForTesting('client/authentication')],
   'ldapsearchsimple': [tu.pathForTesting('client/authentication')],
-  'ldapfailover': tu.pathForTesting('client/ldapfailover')
+  'ldapfailover': tu.pathForTesting('client/ldap')
```

Comment:
I still think the suite does not test "failover" but dual ldap only

Reply:
renamed